### PR TITLE
Z-Wave: add section on command class specific functionalities

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- draft proposal for a section to satisfy this certification requirement:
- _The product documentation MUST point out how to perform each action associated to the minimum end user functionalities required by a Command Class_.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zwave_js.markdown [hunk 1]
```diff
@@ -844,3 +844,24 @@ Z-Wave JS writes details to its logs. To access these logs go to the following.
...
+Covers (called *barrier operator* in Z-Wave terminology) can be opened, closed, and stopped via the corresponding buttons.
+
+![Cover controls](/images/integrations/z-wave/cmd-class-cover.png)
+
+### Binary switch
+Devices can be turned on or off via toggle buttons.
+![Cover controls](/images/integrations/z-wave/cmd-class-binary-switch.png)
+
+### Color switch
+Colors can be dimmed using a slider and defined using a color picker.
```
Comment (AlCalzone) [inline source/_integrations/zwave_js.markdown:874]: Color Switch CC is only concerned with setting the color. For brightness, it refers to Multilevel Switch or Binary Switch.

We may also need to describe how to get to the color wheel.

Comment (AlCalzone) [reply source/_integrations/zwave_js.markdown:874]: Not sure if we should have bullet points or subsections for each of the required actions? In this case those would be:

- set color
- start fading to a different color
- stop fading

Comment (c0ffeeca7) [reply source/_integrations/zwave_js.markdown:874]: I see. I will remove the brightness from this section.

**Comment by c0ffeeca7** [comment]
Change of strategy.

**Pull request closed without merging**